src/index.ts

```diff
@@ -64,7 +65,7 @@ function webVitalsModule () {
   nuxt.options.alias['~vitals-provider'] = provider.runtime
 
   // Workaround for nuxt < 2.16.0
-  nuxt.options.alias.ufo = 'ufo/dist/index.mjs'
+  nuxt.options.alias.ufo = await resolvePath('ufo/dist/index.mjs')
```

Comment:
Since ufo allows `*` exports, we can directly use `require.resolve`:

```suggestion
  nuxt.options.alias.ufo = require.resolve('ufo/dist/index.mjs')
```